src/components/SidePane/HelpContent/chat/admins.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable react/no-unescaped-entities */
+import React from 'react';
+import BulletList from '@components/SidePane/HelpComponents/HelpBulletList';
+import ExpandableHelp from '@components/SidePane/HelpComponents/HelpExpandable';
+import Text from '@components/Text';
+import type {ThemeStyles} from '@styles/index';
+
+function AdminsChatRoom({styles}: {styles: ThemeStyles}) {
+    return (
+        <>
+            <Text style={[styles.textHeadlineH1, styles.mb4]}>#admins</Text>
+            <ExpandableHelp
+                styles={styles}
+                title="Every workspace automatically receives a special #admins chat room. Every admin is automatically added to this room as a member. The #admins room is used for several purposes:"
```

Comment:
Hmm, is it normal to have all these copies here instead of using the translation keys as everywhere else? The same applies to the other files. Not sure if we are planning to do it later

Reply:
The side pane is only displayed when English is selected in the settings, so we decided to hardcode the text in these components.

(Eventually everything will be moved to markdown files so it doesn't matter)

Reply:
Discussed here: https://expensify.slack.com/archives/C07NZ8B1VTQ/p1742827061945449?thread_ts=1742502068.420729&cid=C07NZ8B1VTQ